index.html

```diff
@@ -25,21 +25,26 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
   </head>
   <body>
     <div class="container">
       <div class="nav-container">
         <header>
           <nav class="nav-bar wrap-items">
             <a href="#">Gbenga</a>
-            <img src="./images/Union.png" alt="harmburger" id="drop-menu" />
+            <div class="open-menu">
+              <img src="./images/Union.png" alt="hamburger">
+            </div>
             <ul class="nav-links">
-              <li><a href="#">Porfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
-              <li><a href="#"><img src="./images/mail.png" alt="mail"></a></li>
+              <li class="portfolio-toggle"><a href="#works" class="menu">Porfolio</a></li>
+              <li class="about-toggle"><a href="#about" class="menu">About</a></li>
+              <li class="contact-toggle"><a href="#contact" class="menu">Contact</a></li>
+              <li class="home-icon"><br><br><img src="./images/home-indicator.png" alt="home icon" class="home-icon"></li> 
             </ul>
```

Comment:
## Optional

- [ ] Nice Job for trying to implement a separator line under Portfolio, About, Contact. Please try your best in making it looks like the one in Figma design.